File: src/main/java/com/ject/studytrip/mission/application/dto/MissionsInfo.java
package com.ject.studytrip.mission.application.dto;

import java.util.List;

public record MissionsInfo(List<MissionInfo> missionInfos) {
    public static MissionsInfo of(List<MissionInfo> missionInfos) {
        return new MissionsInfo(missionInfos);
    }
}

File: src/main/kotlin/com/ject/studytrip/mission/application/dto/MissionsInfo.kt
package com.ject.studytrip.mission.application.dto

data class MissionsInfo(
    val missionInfos: List<MissionInfo>,
) {
    companion object {
        @JvmStatic
        fun of(missionInfos: List<MissionInfo>): MissionsInfo = MissionsInfo(missionInfos)
    }
}
